src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -619,12 +620,12 @@ public PrimarySecondaryIPListPair getReplicaListPair(long context, DHTKey key, R
         IPAndPort[]   replicas;
         
         if (debug) {
-            System.out.println("\t\tgetPrimaryReplicas "+ key);
+            Log.fineAsyncf("getPrimaryReplicas ", key);
         }
         // FUTURE - think about improvements
         replicas = ringMaster.getReplicas(key, oqm, ownerQueryOpType);
         if (debug) {
-            System.out.println("\t\t"+ key +"\t"+ IPAndPort.arrayToString(replicas));
+            Log.fineAsyncf("%s \t %s", key , IPAndPort.arrayToString(replicas));
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect